pkg/util/jsonnet.go

```diff
@@ -49,3 +50,39 @@ func UnmarshalConfigurationFromFile(path string, configuration proto.Message) er
 	}
 	return nil
 }
+
+// MarshalExample
+// marshals the  Protobuf message into a json string
+func MarshalExample(msg interface{}) (string, error) {
+
+	if msg == nil {
+		return "", status.Errorf(codes.InvalidArgument, "Unable to marshal nil type")
+	}
+
+	v := reflect.ValueOf(msg)
+
+	initializeMessage(v.Elem().Type(), v.Elem())
+
+	marshaler := &jsonpb.Marshaler{
+		EmitDefaults: true,
+		Indent:       "  ",
+	}
+	jsonString, err := marshaler.MarshalToString(msg.(proto.Message))
+	if err != nil {
+		return jsonString, StatusWrap(err, "Failed to marshal protobuf message")
+	}
+
+	return jsonString, err
+}
+
+func initializeMessage(t reflect.Type, v reflect.Value) {
+	for i := 0; i < v.NumField(); i++ {
+		f := v.Field(i)
+		ft := t.Field(i)
+		if ft.Type.Kind() == reflect.Ptr {
```

Comment:
How does this work in case of recursively defined Protobuf messages? What about fields that are Protobuf implementation details? Should we use [protoreflect](https://godoc.org/google.golang.org/protobuf/reflect/protoreflect) instead?

Reply:
1- Good question, do we have an example of such msg?
2,3 - I don't know protoreflect but I can definitely check.

Reply:
BlobAccessConfiguration is a good example of that. Though right now that won't trigger, because there's a `oneof` inside of it that breaks the loop.